esda/significance.py

```diff
@@ -0,0 +1,164 @@
+import numpy as np
+import warnings
+
+try:
+    from numba import njit
+except (ImportError, ModuleNotFoundError):
+
+    def jit(*dec_args, **dec_kwargs):
+        """
+        decorator mimicking numba.jit
+        """
+
+        def intercepted_function(f, *f_args, **f_kwargs):
+            return f
+
+        return intercepted_function
+
+    njit = jit
```

Comment:
We've been typically doing this as below to avoid reimplementation of decorator in every package but it does not matter much.

```py
try:
    from numba import njit
except (ModuleNotFoundError, ImportError):
    from libpysal.common import jit as njit
```